__tests__/action_single.test.js

```diff
@@ -13,12 +13,16 @@ describe("Single report", function () {
 
   function getInput(key) {
     switch (key) {
-      case "paths":
-        return "./__tests__/__fixtures__/report.xml";
-      case "min-coverage-overall":
+      case `paths`:
+        return `./__tests__/__fixtures__/report.xml`;
+      case `min-coverage-overall`:
         return 45;
       case `min-coverage-changed-files`:
         return 60;
+      case `baseline-paths`:
+        return ``;
+      case `show-pages-links`:
```

Comment:
Will remove: this is now unused

Reply:
Do these strings have to use backticks? No variable substitution or concatenation is going on. While there's no tangible performance hit to using them in practice (max 1% difference, direction depends on the engine (e.g., chrome is faster now with backticks than concatenation)), you're technically sacrificing compatibility with old JS engines.

Reply:
They don't: the convention was mixed in the forked repo so I standardized on one. Is there concern over old JS engines for GH actions, especially for the uncompiled code? We're using string interpolation throughout.